tests/filecheck/dialects/stencil/invalid.mlir

```diff
@@ -105,3 +115,55 @@ builtin.module {
 }
 
  // CHECK: 'stencil.access' expects parent op 'stencil.apply'
+
+// -----
+
+builtin.module {
+  func.func @wrong_return_arity_1d(%in : !stencil.field<[-4,68]xf64>, %bigin : !stencil.field<[-4,68]x[-4,68]xf64>, %out : !stencil.field<[-4,68]xf64>) {
+    %int = "stencil.load"(%in) : (!stencil.field<[-4,68]xf64>) -> !stencil.temp<?xf64>
+    %outt1, %outt2 = "stencil.apply"(%int) ({
+    ^0(%intb : !stencil.temp<?xf64>):
+      %v = "stencil.access"(%intb) {"offset" = #stencil.index<-1>} : (!stencil.temp<?xf64>) -> f64
+      "stencil.return"(%v) : (f64) -> ()
+    }) : (!stencil.temp<?xf64>) -> (!stencil.temp<?xf64>, !stencil.temp<?xf64>)
+    "stencil.store"(%outt1, %out) {"lb" = #stencil.index<0>, "ub" = #stencil.index<68>} : (!stencil.temp<?xf64>, !stencil.field<[-4,68]xf64>) -> ()
+    "func.return"() : () -> ()
+  }
+}
+
+ // CHECK: stencil.return expected 2 operands to match the parent stencil.apply result types, got 1
+
+// -----
+
+builtin.module {
+  func.func @wrong_return_types_1d(%in : !stencil.field<[-4,68]xf64>, %bigin : !stencil.field<[-4,68]x[-4,68]xf64>, %out : !stencil.field<[-4,68]xf64>) {
+    %int = "stencil.load"(%in) : (!stencil.field<[-4,68]xf64>) -> !stencil.temp<?xf64>
+    %outt1, %outt2 = "stencil.apply"(%int) ({
+    ^0(%intb : !stencil.temp<?xf64>):
+      %v = "stencil.access"(%intb) {"offset" = #stencil.index<-1>} : (!stencil.temp<?xf64>) -> f64
+      "stencil.return"(%v, %v) : (f64, f64) -> ()
+    }) : (!stencil.temp<?xf64>) -> (!stencil.temp<?xf64>, !stencil.temp<?xf32>)
+    "stencil.store"(%outt1, %out) {"lb" = #stencil.index<0>, "ub" = #stencil.index<68>} : (!stencil.temp<?xf64>, !stencil.field<[-4,68]xf64>) -> ()
+    "func.return"() : () -> ()
+  }
+}
+
+ // CHECK: stencil.return expected operand types to match the parent stencil.apply result element types.
+
+
+// -----
+
+builtin.module {
+  func.func @different_apply_bounds(%in : !stencil.field<[-4,68]xf64>, %bigin : !stencil.field<[-4,68]x[-4,68]xf64>, %out : !stencil.field<[-4,68]xf64>) {
+    %int = "stencil.load"(%in) : (!stencil.field<[-4,68]xf64>) -> !stencil.temp<?xf64>
+    %outt1, %outt2 = "stencil.apply"(%int) ({
+    ^0(%intb : !stencil.temp<?xf64>):
+      %v = "stencil.access"(%intb) {"offset" = #stencil.index<-1>} : (!stencil.temp<?xf64>) -> f64
+      "stencil.return"(%v, %v) : (f64, f64) -> ()
+    }) : (!stencil.temp<?xf64>) -> (!stencil.temp<?xf64>, !stencil.temp<[0,64]xf64>)
+    "stencil.store"(%outt1, %out) {"lb" = #stencil.index<0>, "ub" = #stencil.index<68>} : (!stencil.temp<?xf64>, !stencil.field<[-4,68]xf64>) -> ()
+    "func.return"() : () -> ()
+  }
+}
+
+// CHECK: Expected all output types bounds to be equals.
```

Comment:
```suggestion
// CHECK: Expected all output types bounds to be equals.

```

Reply:
:scream: 
Weren't those supposed to be catched by CI now? I might push a lot of those otherwise... I'm amazed how often I forget them. Reminds me of the early days' missing `;`s in C..

Reply:
You might want to install the pre-commit hook

Reply:
Oh, that's what.. thanks!